Error while building

Hi, I used the commande ``git clone --recursive https://github.com/WerWolv/Tesla-Template/" To clone the repo and when I want to build the example I have this error. What's the problem ?

```
main.cpp
C:/Users/geyre/Desktop/peerplay_overlay/source/main.cpp:35:58: error: 'touchPosition' has not been declared
   35 |     virtual bool handleInput(u64 keysDown, u64 keysHeld, touchPosition touchInput, JoystickPosition leftJoyStick, JoystickPosition rightJoyStick) override {
      |                                                          ^~~~~~~~~~~~~
C:/Users/geyre/Desktop/peerplay_overlay/source/main.cpp:35:84: error: 'JoystickPosition' has not been declared
   35 |     virtual bool handleInput(u64 keysDown, u64 keysHeld, touchPosition touchInput, JoystickPosition leftJoyStick, JoystickPosition rightJoyStick) override {
      |                                                                                    ^~~~~~~~~~~~~~~~
C:/Users/geyre/Desktop/peerplay_overlay/source/main.cpp:35:115: error: 'JoystickPosition' has not been declared
   35 |     virtual bool handleInput(u64 keysDown, u64 keysHeld, touchPosition touchInput, JoystickPosition leftJoyStick, JoystickPosition rightJoyStick) override {
      |                                                                                                                   ^~~~~~~~~~~~~~~~
C:/Users/geyre/Desktop/peerplay_overlay/source/main.cpp:35:18: error: 'virtual bool GuiTest::handleInput(u64, u64, int, int, int)' marked 'override', but does not override
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -1,0 +1,16 @@
+name: build
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: devkitpro/devkita64:latest
+
+    steps:
+    - name: Checkout 🛎️
+      uses: actions/checkout@master
+      with:
+        submodules: recursive
+
+    - name: Build
+      run: make -j2

--- a/source/main.cpp
+++ b/source/main.cpp
@@ -21,7 +21,7 @@ public:
 
         // Add the list to the frame for it to be drawn
         frame->setContent(list);
-        
+
         // Return the frame to have it become the top level element of this Gui
         return frame;
     }
@@ -32,8 +32,8 @@ public:
     }
 
     // Called once every frame to handle inputs not handled by other UI elements
-    virtual bool handleInput(u64 keysDown, u64 keysHeld, touchPosition touchInput, JoystickPosition leftJoyStick, JoystickPosition rightJoyStick) override {
-        return false;   // Return true here to singal the inputs have been consumed
+    virtual bool handleInput(u64 keysDown, u64 keysHeld, const HidTouchState &touchPos, HidAnalogStickState joyStickPosLeft, HidAnalogStickState joyStickPosRight) override {
+        return false;   // Return true here to signal the inputs have been consumed
     }
 };
 

